remoteappmanager/docker/container_manager.py

```diff
@@ -165,7 +158,7 @@ def _start_container(self, user_name, image_name, volumes):
         # build the dictionary of keyword arguments for create_container
         container_name = _generate_container_name("remoteexec",
                                                   user_name,
-                                                  image_name)
+                                                  mapping_id)
```

Comment:
The key question here is, can a user have multiple container instances for the same (application, policy) tuple?